assets/styles/style.css

```diff
@@ -0,0 +1,291 @@
+body {
+  margin: 0;
+  background: #1CB5E2;
+  background-image: -webkit-radial-gradient(bottom, circle cover, #1CB5E2 0%, #000060 100%);
+  overflow-x: hidden;
+  overflow-y: hidden;
+}
+
+.main {
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-content: center;
+  align-items: center;
```

Comment:
Please check if all three of these are needed. Maybe one of them is redundant.

Reply:
I'll fix this. Meanwhile, can you suggest some good gradients? The whitish tones could be the result of using radial gradient.